swagger-def/definitions/PPMShipment.yaml

```
@@ -101,34 +111,39 @@ properties:
     x-nullable: true
     x-omitempty: false
   sitExpected:
+    description: Captures whether the PPM is expected to need put into storage at the origin or destination.
```

The wording feels off.
Maybe 
```suggestion
    description: Captures whether the PPM is expected to need storage at the origin or destination.
```

---

Yea I went back and forth with that one, I was trying to capture that storage is a different place but it still feels awkward.

---

Do we need to capture that? Wouldn't something like `Capture whether there is the intent to store some goods in temporary storage` suffice since the sitLocation deals with the determination of Origin/Destination. We could also add a sentence indicating that this is related to the sitLocation field.

---

Hmm good point let me throw in the relationship there too.